step/src/main/java/com/google/sps/Message.java

```diff
@@ -25,12 +25,14 @@
   private final String recipientID; 
   private final String text; 
   private final long timestamp;
+  private final String timeStamp;
 
-  public Message(String senderID, String recipientID, String text, long timestamp) {
+  public Message(String senderID, String recipientID, String text, long timestamp, String timeStamp) {
       this.senderID = senderID;
       this.recipientID = recipientID;
       this.text = text;
       this.timestamp = timestamp;
+      this.timeStamp = timeStamp;
```

Comment:
I store them in 2 different variables because the "long" one is used to rank the sequence of the messages displayed, the "String" one is called in jsp files to get access to the data. I understand that it might confuse people when the variable names look pretty similar. If you have any better idea, then I can make changes :)